res/layout/preview_video_fragment.xml

```diff
@@ -33,27 +33,15 @@
         android:id="@+id/visual_area"
         android:layout_width="match_parent"
         android:layout_height="0dp"
-        android:layout_above="@+id/media_controller"
         android:layout_alignParentTop="true"
         >
 
-        <VideoView
-            android:id="@+id/video_preview"
+        <com.google.android.exoplayer2.ui.SimpleExoPlayerView android:id="@+id/video_preview_fragment"
```

Comment:
Mmmm, this file is *not* being used anymore. Both the ```PreviewVideoFragment``` and ```PreviewActivity``` use ```res/layout/video_player.xml```. Please, remove one of them